scripts/fork/helpers/impersonateNamedAccounts.ts

```diff
@@ -0,0 +1,11 @@
+import chalk from 'chalk';
+import hre from "hardhat";
+
+async function impersonateNamedAccounts(ptOperations: string){
```

Comment:
`impersonatePtOperations` would be more precise. Here `impersonateNamedAccounts` is generic but we expect a `ptOperations` address.